compiler-core/src/erlang.rs

```diff
@@ -447,7 +447,10 @@ fn atom_pattern() -> &'static Regex {
 }
 
 fn atom(value: &str) -> Document<'_> {
-    if is_erlang_reserved_word(value) {
+    if is_erlang_existing_function_name(value) {
```

Comment:
Rather than always change atom names it should only be functions that get their name changed.